js/data.js

```diff
@@ -74,5 +74,4 @@ const createData = function () {
 };
 
 const createArrayData = function () {return Array.from({length: SIMULATION_ARRAY_DATA_COUNT}, createData);};
```

Comment:
Лучше не использовать array если есть привязка к данным, тут функция похожа на createOffers например плюс нужно одинако определять функции, тоже самое с createData, лучше назвать более явно, например createOffer. Data -- это просто какие-то данные, тут ты явно и конкретно создаешь данные предложения.
```suggestion
function createOffers  () {return Array.from({length: SIMULATION_ARRAY_DATA_COUNT}, createData);};
```